checker/src/org/checkerframework/checker/regex/RegexUtil.java

```diff
@@ -2,16 +2,16 @@
 
 package org.checkerframework.checker.regex;
 
-import java.util.regex.Pattern;
```

Comment:
This needs to be synced to https://github.com/mernst/plume-lib/blob/master/java/src/plume/RegexUtil.java.